app/controllers/admins_controller.rb

```diff
@@ -64,9 +60,8 @@ def verify
     if voter.save
       send_email = params[:send_email] == "true"
       if send_email
-        # Will need to be replaced with deliver_now
         begin
-          UserMailer.voter_verified(voter, event_year).deliver
+          UserMailer.voter_verified(voter, event_year).deliver_now
```

Comment:
I'm not familiar with how your prod server is set up. If it is still using rails 4.1.8 then the bundler-audit branch didn't do its job correctly.

Can you try running `bundle exec rails -v`, that uses the version bundle thinks it should be using, not the first thing called `rails` in your PATH.